From 30d5b5635556e3a3fd68aa1a8a47c2480229ae8c Mon Sep 17 00:00:00 2001
From: Varun Sharma <vsharma@habana.ai>
Date: Thu, 05 Oct 2023 12:03:27 +0300
Subject: [PATCH] [SW-66365] security: add hl_fit_if.h specs header

This patch adds hl_fit_if.h include, as this is needed for
specs patch which moves fit specific macro's & data structures
to common file: hl_fit_if.h. Including this preemptively
will make sure ci passes for specs patch.

Change-Id: I31e2b24aa3c34121112f1c99d552f669fcbe274c
Signed-off-by: Varun Sharma <vsharma@habana.ai>
---

diff --git a/arch/gaudi2/common/security.c b/arch/gaudi2/common/security.c
index 566f480..4e53c46 100644
--- a/arch/gaudi2/common/security.c
+++ b/arch/gaudi2/common/security.c
@@ -58,6 +58,7 @@
 #include <pll.h>
 #include <gaudi2_fw_if.h>
 #include <hl_sec_if.h>
+#include <hl_fit_if.h>
 
 #include "common.h"
 
